lib/default-config.js

```diff
@@ -3,7 +3,7 @@ const { SORT_BY, SORT_DIRECTIONS } = require('./sort-pull-requests')
 const DEFAULT_CONFIG = Object.freeze({
   'name-template': '',
   'tag-template': '',
-  'change-template': `* $TITLE (#$NUMBER) @$AUTHOR`,
+  'change-template': `* $TITLE (#$NUMBER) @$AUTHOR $DESC`,
```

Comment:
Should not be added to the default, as I think this is optional.

Reply:
Can you suggest an alternate way to add optional description in this template?

Reply:
Simply remove it, and you inside your `release-drafter.yml` file you can override the `change-template` to include description.
Changing the default is a breaking change.

Reply:
But I want to add 'PR Description' which comes from api in body tag.

Reply:
@jetersen Oh yes, it's working. Thanks!